rotkehlchen/premium/premium.py

```diff
@@ -406,6 +414,7 @@ def watcher_query(
             log.error(msg)
             raise RemoteError(msg) from e
 
+        check_response_status_code(response)
```

Comment:
Can you tell me here what was the status code of the response from the server? Was it non-200? Are you sure? If yes and your fix is abstracting this it should be used in all other places not only here.

I will amend the PR.